### PR TITLE
Added Laravel 5.7 compatibility

### LINKED ISSUE #83: install issue in laravel 5.7
## Detailed description

when i try to install laravel-page-speed using composer, i ll see the following message:

Installation failed, reverting ./composer.json to its original content.

## Your environment

Include as many relevant details about the environment you experienced the bug in and how to reproduce it.

* PHP 7.2:
* window 10

### PR DESCRIPTION
**Pull request by tswestendorp**
## Description

Add 5.7.* to the composer.json to it can be installed with Laravel 5.7

## Motivation and context

Allows you to use this package with Laravel 5.6

fixes #83

## How has this been tested?

`composer test`

## Types of changes

What types of changes does your code introduce? Put an `x` in all the boxes that apply:
- [ ] Bug fix (non-breaking change which fixes an issue)
- [X] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

Go over all the following points, and put an `x` in all the boxes that apply.

Please, please, please, don't send your pull request until all of the boxes are ticked. Once your pull request is created, it will trigger a build on our [continuous integration](http://www.phptherightway.com/#continuous-integration) server to make sure your [tests and code style pass](https://help.github.com/articles/about-required-status-checks/).

- [x] I have read the **[CONTRIBUTING](CONTRIBUTING.md)** document.
- [x] My pull request addresses exactly one patch/feature.
- [x] I have created a branch for this patch/feature.
- [x] Each individual commit in the pull request is meaningful.
- [ ] I have added tests to cover my changes.
- [x] If my change requires a change to the documentation, I have updated it accordingly.

If you're unsure about any of these, don't hesitate to ask. We're here to help!

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -17,7 +17,7 @@
   ],
   "require": {
     "php": "^5.6 || ^7.0",
-    "illuminate/support": "5.3.* || 5.4.* || 5.5.* || 5.6.*"
+    "illuminate/support": "5.3.* || 5.4.* || 5.5.* || 5.6.* || 5.7.*"
```
Comment (hofmannsven) [inline composer.json]: 👍

Comment (aboda1986) [reply composer.json]: what is the problem with add support to laravel 5.7?

Comment (joaorobertopb) [reply composer.json]: No problem!

**Comment by tswestendorp** [comment]
@joaorobertopb looks like Packagist somehow didn't pick up 1.8.8, something you can do about this?

**Comment by joaorobertopb** [comment]
Hmm...

@tswestendorp I do not have access to the Packagist of this repository...

@renatomarinho, can you help us, please?

**Pull request merged**